src/components/analytics/EndOfDay.vue

```diff
@@ -125,6 +136,9 @@ export default defineComponent({
         company: this.company,
       });
     },
+    printDiv(div_id: string, title: string, style?: string) {
```

Comment:
call printDiv directly, instead of creating function with same name inside component.